.circleci/config.yml

```diff
@@ -459,19 +459,19 @@ jobs:
       - setup_remote_docker
       - run:
           name: Test installing packages on target systems
-          command: make -f dockerfiles/verify_packages/Makefile
+          command: make -j 2 -f dockerfiles/verify_packages/Makefile
 
 workflows:
   version: 2
   build_packages:
     jobs:
       - "5.6 20131106": &BUILD_PACKAGE_FORKFLOW
-          filters:
-            tags:
-              only: /(^build$)|(^[v]?[0-9]+(\.[0-9]+)*$)/
-            branches:
-              # Always build on master
-              ignore: /^(?!master$).*/
+          filters: {}
```

Comment:
is this intended?

Reply:
No - its just temporary - to unblock the CI. I'll remove it before merging